packages/localizations/src/de-DE.ts

```diff
@@ -807,7 +813,7 @@ export const deDE: LocalizationResource = {
     mobileButton__menu: 'Menü',
     navbar: {
       account: 'Profil',
-      billing: undefined,
+      billing: 'Billing',
```

Comment:
Note to myself: Ask team if we want to translate this

Reply:
```suggestion
      billing: 'Abrechnung',
```

We can translate it 👍